docs/usage/verticles/DeployerVerticle.md

```diff
@@ -0,0 +1,42 @@
+# DeployerVerticle
```

Comment:
Agreed, we should write a doc about NeonBee modules. We can do it in a separate change and then link the sections referring to it. Otherwise, I fear this one becomes unreviewable.